README.adoc

```diff
@@ -0,0 +1,97 @@
+= Interrupt policy
+
+ifdef::env-github[]
+image:https://img.shields.io/static/v1?label=Available%20at&message=Gravitee.io&color=1EC9D2["Gravitee.io", link="https://download.gravitee.io/#graviteeio-apim/plugins/policies/gravitee-policy-interrupt/"]
+image:https://img.shields.io/badge/License-Apache%202.0-blue.svg["License", link="https://github.com/gravitee-io/gravitee-policy-interrupt/blob/master/LICENSE.txt"]
+image:https://img.shields.io/badge/semantic--release-conventional%20commits-e10079?logo=semantic-release["Releases", link="https://github.com/gravitee-io/gravitee-policy-interrupt/releases"]
+image:https://circleci.com/gh/gravitee-io/gravitee-policy-interrupt.svg?style=svg["CircleCI", link="https://circleci.com/gh/gravitee-io/gravitee-policy-interrupt"]
+endif::[]
+
+== Phase
+
+[cols="4*", options="header"]
+|===
+^|onRequest
+^|onResponse
+^|onRequestContent
+^|onResponseContent
+
+^.^| X
+^.^|
+^.^| X
+^.^|
+
+|===
+
+== Description
+
+`Interrupt` policy can be used to break the entire request processing in case of a condition (to define on the policy.
+By default, if no policy condition is defined, the policy will always break request processing).
+
+Breaking the request processing means that no more policy will be executed and no endpoint will be called by the gateway.
+
+By default, the policy will return a response payload to the consumer which is containing the `message` (see the
+configuration section).
+
+If you want to override this standard response from the policy, you can define an `errorKey` which will be then be used to
+define a Response Template.
+
+
+== Compatibility with APIM
+
+|===
+|Plugin version | APIM version
+
+|1.x and upper                  | 3.10.x to latest
```

Comment:
In 3.10.x, we rely on bom 1.5 and expression-language 1.6.3.
But in the pom of this policy, we rely on bom 2.5 and EL 1.9.2.

Are we sure this policy will be compatible with 3.10.x ?

Reply:
Ok, let me think about it because I wanted to benefit from the new Test SDK framework, and so, if I change the version, I would not be able to run them anymore....
But, at the same time, this policy can be used with Gravitee.... 1.x...

Reply:
If the policy works with all version of gravitee, then forget my comment. It was just like a reminder to check compatibility.

Reply:
I've putted 3.10 just because that is the oldest supported version.